orchestrator/services/activity_service.py

```
@@ -119,8 +119,61 @@ def get_feed(
+            rows = self.db.execute(
+                text(
+                    """
+                    SELECT DISTINCT ON (run_id) run_id, event_type, created_at
+                    FROM orchestration_events
+                    WHERE run_id::text = ANY(:run_ids)
+                    ORDER BY run_id, created_at DESC
+                    """
+                ),
+                {"run_ids": run_ids},
+            ).fetchall()
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)